### PR TITLE
Removed the recursive call to Request.process() which caused stack overflows

### PR DESCRIPTION
**Pull request by TotalKrill**
Stack overflow would occur when using changefeeds, which are kind of the best feature with

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -167,3 +169,16 @@ macro with_args($cmd: ident, $args: ident) {{
...
+
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    //Just perform some basic integration tests against a server
+    #[test]
+    fn it_works() {
+        assert_eq!(2 + 2, 4);
```
Comment (rushmorem) [inline src/lib.rs]: Did you mean to add a proper test here?

Comment (TotalKrill) [reply src/lib.rs]: Ah yes, that was my plan. I was about start just adding basic connect/write/read tests, to get rid of running them manually on the side. Basically small applications running against a local server.

I must have commited that one by mistake, sorry.

**Comment by rushmorem** [comment]
LGTM. Thanks!

**Comment by rushmorem** [comment]
I've released `v0.0.26` with your latest changes. Thanks!

**Pull request merged**